platforms/ide/tooling-api/src/main/java/org/gradle/tooling/events/problems/ProblemsSummaries.java

```diff
@@ -14,17 +14,24 @@
  * limitations under the License.
  */
 
-package org.gradle.tooling.internal.protocol;
+package org.gradle.tooling.events.problems;
 
-import org.gradle.api.NonNullApi;
-import org.gradle.tooling.internal.protocol.problem.InternalProblemDetailsVersion2;
+import org.gradle.api.Incubating;
 
 import java.util.List;
 
-@NonNullApi
-public interface InternalProblemAggregationVersion3 extends InternalProblemDetailsVersion2 {
-
-    InternalProblemDefinition getProblemDefinition();
+/**
+ * All problems summaries
```

Comment:
💄 Missing . at the end of the sentence. Add @see annotation to ProblemsSummariesEvent.